.golangci.yml

```diff
@@ -4,11 +4,6 @@ run:
   build-tags:
     - tools
     - e2e
```

Comment:
Q: Why was this moved from here to exclude-files? New config file format in 1.60.2?

Reply:
(mostly wondering what the difference is between configuration here or in exclude-files otherwise)

Reply:
I got a warning `skip-files` is deprecated

Reply:
Thx, good enough :)